src/modeler/AutoRest.Swagger/Validation/SecurityDefinitionsStructureValidation.cs

```diff
@@ -24,6 +24,11 @@ public class SecurityDefinitionsStructureValidation : TypedRule<Dictionary<strin
         public override ValidationCategory ValidationCategory => ValidationCategory.SDKViolation;
 
         /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public override ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.ServiceImpactingChanges;
```

Comment:
So if I understand correctly, this just defines the mechanics of communicating with the service. If the service allows communication irrespective of the security definitions provided, this really shouldn't affect either the service or the SDK generated. tl;dr ValidationChangesImpact must be None.
Thoughts @veronicagg ?